apollo-router/src/plugins/telemetry/metrics/apollo/studio.rs

```diff
@@ -198,19 +201,32 @@ pub(crate) struct FieldStat {
     errors_count: u64,
     estimated_execution_count: f64,
     requests_with_errors_count: u64,
-    latency: DurationHistogram,
+    latency: DurationHistogram<f64>,
```

Comment:
probably worth a comment to explain why these are floats (and mention that they still get rounded when converted to proto)?